src/System.ComponentModel.Primitives/src/System/ComponentModel/DesignOnlyAttribute.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel
+{
+    /// <devdoc>
+    ///    <para>Specifies whether a property can only be set at
+    ///       design time.</para>
+    /// </devdoc>
+    [AttributeUsage(AttributeTargets.All)]
+    public sealed class DesignOnlyAttribute : Attribute, IIsDefaultAttribute
+    {
+        private readonly bool _isDesignOnly = false;
```

Comment:
Nit: as earlier, this gets overwritten by the ctor anyway (and false is the default), so there's no need to initialize it here.  I'll stop commenting on this, assuming it occurs again in other attributes later in the PR.